ddtrace/contrib/internal/google_adk/__init__.py

```diff
@@ -0,0 +1,46 @@
+"""
+The Google ADK integration instruments the Google ADK Python SDK to trace Agent requests.
```

Comment:
This description makes it sound like we instrument calls made to the Trace Agent requests, but maybe you mean something like:
```suggestion
The Google ADK integration instruments the Google ADK Python SDK to create spans for Agent requests.
```